bindings/rust/s2n-tls-sys/build.rs

```diff
@@ -104,6 +104,10 @@ fn build_vendored() {
     // fortify source is only available in release mode
     if env("PROFILE") == "release" {
         build.define("_FORTIFY_SOURCE", "2");
```

Comment:
Might want to move the fortify comment down to line 106, now that this block does more than that